### PR TITLE
Update Write_ChckOutLst.m for AeroDyn multi-rotor

### PR DESCRIPTION
**Pull request by hkross**
Update Write_ChckOutLst.m to correctly generate I/O code for AeroDyn, considering changes needed to support multiple rotors (i.e., variables passing and declarations).

### CODE REVIEW

**Review by ebranlard — APPROVED** [review]

**Comment by ebranlard** [comment]
Hi Hannah, I'm waiting for the branches to be merged in OpenFAST before merging your pull request in that repo, I hope that 's fine with you!

**Comment by hkross** [comment]
> Hi Hannah, I'm waiting for the branches to be merged in OpenFAST before merging your pull request in that repo, I hope that 's fine with you!

Yep, no problem!

**Comment by hkross** [comment]
I think all relevant branches have been merged into openfast/dev, so this can probably be merged now as well.

**Pull request merged**
